src/main/scala/devices/wdt/TLWDTPeriphery.scala

```diff
@@ -0,0 +1,49 @@
+package sifive.blocks.devices.wdt
+
+import chisel3._
+
+import org.chipsalliance.cde.config.{Field, Parameters}
+import freechips.rocketchip.diplomacy._
+import freechips.rocketchip.interrupts._
+import freechips.rocketchip.prci._
+import freechips.rocketchip.regmapper._
+import freechips.rocketchip.tilelink._
+import freechips.rocketchip.subsystem._
+import freechips.rocketchip.devices.tilelink._
+import freechips.rocketchip.util._
+
+import sifive.blocks.util._
+import sifive.blocks.devices.mockaon.WatchdogTimer
+
+case object PeripheryWDTKey extends Field[Seq[WDTParams]](Nil)
+
+trait HasPeripheryWDT { this: BaseSubsystem =>
+    val wdtNodes = p(PeripheryWDTKey).map { ps =>
+        WDTAttachParams(ps).attachTo(this).ioNode.makeSink()
```

Comment:
Can you fix this file to use 2-space indentation?

Reply:
Ok changed